pkg/services/featuremgmt/registry.go

```diff
@@ -1353,6 +1353,13 @@ var (
 			Owner:           grafanaAlertingSquad,
 			RequiresRestart: true,
 		},
+		{
+			Name:        "restoreDashboards",
```

Comment:
we already have `dashboardRestore` which we're using for the backend. We should be more explicit about this new feature flag that's just for the frontend.

Reply:
yeah, I've just found that. On my way to change it. I will also add that it is frontend only to the description 👌

Reply:
Make sure that the old name doesn't persist in the `pkg/services/featuremgmt/toggles_gen.json` file

Reply:
I thought this was automatic after running the tests again. Should I just remove it?